server/utils/getCaseListFilters.js

```diff
@@ -1,7 +1,7 @@
 module.exports = (caseListData, selectedFilters) => {
   const availableProbationStatuses = [...new Set(caseListData.map(item => item.probationStatus))]
   const probationStatuses = []
-  const statusOrder = ['Current', 'Previously known', 'No record']
+  const statusOrder = ['Current', 'Previously known', 'No record', 'Pre-sentence record']
```

Comment:
The order should be `['Current', 'Pre-sentence record', 'Previously known', 'No record']`